about.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- font-awesome cdn -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <!-- css -->
+  <link rel="stylesheet" href="./css/style.css">
+  <!-- javascript -->
+  <script src="./javascript/navbar.js" defer></script>
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <div class="socials">
+      <ul>
+        <li>
+          <i class="fa-brands fa-facebook-f"></i>
+        </li>
+        <li>
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li>
+          English
+        </li>
+        <li>
+          My Page
+        </li>
+        <li>
+          Logout
+        </li>
+      </ul>
+    </div>
+    <nav>
+      <ul class="navbar">
+        <li class="logo">
+          <a href="index.html">
+            <img src="./images/blockchain-logo.png" alt="Logo" class="navbar-logo">
+          </a>
+        </li>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li>
+          <a href="#">Sponsor</a>
+        </li>
+        <li>
+          <a href="#">News</a>
+        </li>
+        <li>
+          <a href="#">
+            <button class="join-button" type="button">Block Summit</button>
+          </a>
+        </li>
+      </ul>
+      <div class="hamburger-menu-icon">
+        <i class="fa-solid fa-bars"></i>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="about">
+      <div id="about-intro">
+        <div id="hero-about">
+          <p class="hero-greeting">
+            "Hello! Blockchain Community"
+          </p>
+          <h1 id="hero-heading">
+            BLOCKCHAIN ON-RAMP GLOBAL SUMMIT 2023
+          </h1>
```

Comment:
- [ ]  Kindlt 😃 try here to decrease the font size of this heading to be like the Behance I see that it is bigger than it,so please try to decrease it to be like the Behance 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225303889-0221ddd2-0fb7-4558-819d-03eecff033f6.png)   |   ![image](https://user-images.githubusercontent.com/83360636/225303849-c6f052d2-69b9-40fc-b289-2daa7b690d2c.png) |